app/models/site_banner.rb

```diff
@@ -28,6 +29,7 @@ def donation_amounts_in_scope
     errors.add(:base, "Must have #{NUMBER_OF_DONATION_AMOUNTS} donation amounts") if donation_amounts.count != NUMBER_OF_DONATION_AMOUNTS
     errors.add(:base, "A donation amount is zero or negative") if donation_amounts.select { |donation_amount| donation_amount <= 0}.count > 0
```

Comment:
We're already checking this by validating the approved tip presets right? Can we remove this line?